packages/api/src/external/fhir/shared/hydrate-bundle.ts

```diff
@@ -14,7 +15,7 @@ export function hydrateBundle(
   fhirBundleDestinationKey: string
 ): ValidBundle {
   const docExtension: Extension = {
-    url: "https://public.metriport.com/fhir/StructureDefinition/doc-id-extension.json",
+    url: DOC_ID_EXTENSION_URL,
     valueString: fhirBundleDestinationKey,
   };
```

Comment:
There's a function to build this - `buildDocIdFHIRExtension()`, let's use it?

Reply:
BTW, that fn doesn't set the `valueCoding` of the extension - while the `dataSourceExtensionDefaults` does. Shouldn't it?

Reply:
No, i think it's fine that it doesn't use `valueCoding`. Extensions are allowed to have values of different types, and we have this extension referenced in this format in a bunch of places